spec/swagger/definitions/hand_coded_paths_v1.yaml

```diff
@@ -93,8 +93,8 @@
         enum:
         - asc
         - desc
-    - $ref: pagination_parameters.yaml#/Page
-    - $ref: pagination_parameters.yaml#/PerPage
+    - $ref: /pagination_parameters.yaml#/Page
```

Comment:
I guess the prefix is necessary?

Reply:
Actually no, this was a "partial Undo" 😄